README.md

```diff
@@ -141,38 +144,29 @@ class PlaidEventContainer extends React.Component {
     }
   }
 
-  render() {
-    return (
-      <PlaidLink
-        token={<#GENERATED_LINK_TOKEN#>}
-        onSuccess={data => console.log('success: ', data)}
-        onExit={data => console.log('exit: ', data)}
-      >
-        <Text>Add Account</Text>
-      </PlaidLink>
-    );
-  }
+  ...
 }
 ```
 
 You can also use the `usePlaidEmitter` hook in react functional components:
 
 ```javascript
-  usePlaidEmitter((event) => {
+  usePlaidEmitter((event: LinkEvent) => {
```

Comment:
Something I struggled with is: where to register the hook?
Should we add a bit more context to this sample code?